### ISSUE TITLE
Sourcemap resources are missing from build

### ISSUE THREAD

**Issue by jamesmacwhite** [morpheus65535/bazarr#1466]
**Describe the bug**

I'm running Bazarr through a Apache Reverse Proxy under a subdomain with the path /bazarr. It appears sourcemap resources are not loading correctly with the reverse proxy setup. Everything else works fine.

```
DevTools failed to load source map: Could not load content for https://sub.domain.com/bazarr/static/css/2.ac9f8359.chunk.css.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://sub.domain.com/bazarr/static/css/main.2f23c279.chunk.css.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://sub.domain.com/bazarr/static/js/2.5a10db6f.chunk.js.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://sub.domain.com/bazarr/static/js/main.9f916001.chunk.js.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

Browsing the resource path asks for basic authentication (which is enabled) and then redirects to the ProxyPass domain set, rather than the external subdomain.

**Software (please complete the following information):**

 - Bazarr: [e.g. v 0.6.1] v0.9.5
 - OS: Windows Server 2016 Essentials

**Comment by morpheus65535** [comment]
We're going to release 0.9.6 in the upcoming week. Can you test it once it's release and confirm you still have the issue?

**Comment by jamesmacwhite** [comment]
@morpheus65535 Thank you, I will report back once v0.9.6 is merged into the stable branch.

**Comment by morpheus65535** [comment]
I've just released 0.9.6 to stable. You can now test it.

**Comment by jamesmacwhite** [comment]
I have now had the time to test it, unfortunately, still present on 0.9.6

```
DevTools failed to load source map: Could not load content for https://proxy.domain.com/bazarr/static/css/main.21dea0b3.chunk.css.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://proxy.domain.com/bazarr/static/css/2.ac9f8359.chunk.css.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://proxy.domain.com/bazarr/static/js/main.01a2b3c5.chunk.js.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load source map: Could not load content for https://proxy.domain.com/bazarr/static/js/2.b1d1fcea.chunk.js.map: HTTP error: status code 401, net::ERR_HTTP_RESPONSE_CODE_FAILURE
```

**Comment by jamesmacwhite** [comment]
I think what might be happening is the sourcemap files themselves are being served, if I temporarily disable authentication, I then get:

```
DevTools failed to load source map: Could not parse content for https://proxy.domain.com/bazarr/static/js/2.b1d1fcea.chunk.js.map: Unexpected token < in JSON at position 0
DevTools failed to load source map: Could not parse content for https://proxy.domain.com/bazarr/static/css/main.21dea0b3.chunk.css.map: Unexpected token < in JSON at position 0
DevTools failed to load source map: Could not parse content for https://proxy.domain.com/bazarr/static/css/2.ac9f8359.chunk.css.map: Unexpected token < in JSON at position 0
DevTools failed to load source map: Could not parse content for https://proxy.domain.com/bazarr/static/js/main.01a2b3c5.chunk.js.map: Unexpected token < in JSON at position 0
```

The actual js.map files aren't actually being served, it just loads the UI at /bazarr/series

**Comment by morpheus65535** [comment]
@LASER-Yi any idea why the .map file aren't build? I can confirm the same behavior (`Unexpected token < in JSON at position 0`).

**Comment by jamesmacwhite** [comment]
Could it be the fact that the static assets no longer exist since the web UI refactor

If you make a request to something that doesn't exist under the static path: https://github.com/morpheus65535/bazarr/tree/master/frontend/public/static

i.e proxy.domain.com/bazarr/static/css/notreal.css, it looks like the behaviour is to go to /bazarr/series.

It suggests the map files have been removed, they were originally present under an old path in the source tree.  Maybe since the Web UI React refactor work?

**Comment by morpheus65535** [comment]
Every request that result in a 404 get redirected to /series or /movies if Sonarr integration is disabled.

The issue seems to be that map files aren't included in bazarr.zip asset.

**Comment by jamesmacwhite** [comment]
Ah OK. Makes sense. In addition, this might have also revealed another bug with the redirect logic. It doesn't take  into account a reverse proxy. In my case the redirect location is the domain and 6767 address, not the reverse proxy address.

**Comment by morpheus65535** [comment]
Subdomain and subdirectory proxying is working as expected. You can check linuxserver SWAG proxy-conf for nginx examples.

**Comment by jamesmacwhite** [comment]
It does indeed work fine, I'm using Apache. What I'm pointing out is the redirect if 404 logic, is using the domain and path of what has been configured to reverse proxy, not the actual reverse proxy itself in this case.

Path being proxied: http://domain.com:6767/bazarr/
Actual reverse proxy external: http://proxy.domain.com/bazarr

As these paths currently 404, it causes a redirect to the configured proxy path, not the URL being used to access the proxied app in this case.

**Comment by morpheus65535** [comment]
Ok I get what you means but as Bazarr isn't aware of you subdomain proxy configuration, you'll have to do some url rewrite on proxy side.

**Comment by LASER-Yi** [comment]
> @LASER-Yi any idea why the .map file aren't build? I can confirm the same behavior (`Unexpected token < in JSON at position 0`).

Those files are dropped when zipping

**Comment by jamesmacwhite** [comment]
No worries, the 404 redirect shouldn't happen anyway once the sourcemap files are present.

I can see that a commit was made to not exclude the sourcemap files, so they are part of the release https://github.com/morpheus65535/bazarr/commit/117da38fe6c98b627ac2edd5bf0b7beab6336bc4

I will grab them from a beta and copy them into my install dir. On the next major release they'll be there already.
